src/ovp_pipeline/commands/evidence_verify.py

```
@@ -0,0 +1,315 @@
+    if cutoff_text:
+        where.append("(verified_at = '' OR verified_at < ?)")
+        params.append(cutoff_text)
+    where_sql = ("WHERE " + " AND ".join(where)) if where else ""
+    columns = (
+        "pack, claim_id, source_slug, evidence_kind, quote_text, "
+        "locator, content_hash, retrieval_context, status, verified_at"
+        if table == "claim_evidence"
+        else "pack, source_object_id, target_object_id, relation_type, evidence_source_slug, "
+        "locator, content_hash, retrieval_context, status, verified_at"
+    )
+    return list(conn.execute(f"SELECT {columns} FROM {table} {where_sql}", params).fetchall())
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)